avogadro/io/cjsonformat.cpp

```diff
@@ -257,6 +257,17 @@ bool CjsonFormat::read(std::istream& file, Molecule& molecule)
         for (unsigned int i = 0; i < moCoefficients.size(); ++i)
           coeffs.push_back(static_cast<double>(moCoefficients[i]));
         basis->setMolecularOrbitals(coeffs);
+      } else if (isNumericArray(moCoefficientsA) &&
+                 isNumericArray(moCoefficientsB) &&
+                 moCoefficientsA.size() == moCoefficientsB.size()) {
```

Comment:
I have a nagging feeling that some programs may not keep this last equivalence (e.g., # of alpha and # of beta orbitals may be different)

Reply:
I wondered, I can fix the reader to be a little more relaxed/do two loops...